core/data/pod/value.go

```diff
@@ -145,6 +145,11 @@ func NewValue(val interface{}) *Value {
 	return nil
 }
 
+// NewHandle returns a uint64 as a pod.Type_handle Value.
```

Comment:
If I understand correctly, this is intended for Vulkan handles. It may be worth highlighting this in the function comment.